src/repositories/raweventsRepository.ts

```diff
@@ -0,0 +1,28 @@
+let pgp = require('pg-promise');
+
+const connections = [];
+const dbName = process.env.DBPGNAME;
+const dbUser = process.env.DBPGUSER;
+const dbPassword = process.env.DBPGPASSWORD;
+const dbHost = process.env.DBPGHOST;
+const dbPort = process.env.DBPGPORT;
+const dbc = pgp({ capSQL: true});
+      console.log(`Opening connection to: ${dbName}, host is: ${dbHost}`);
+
+      const connectionString = `postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}/${dbName}?ssl=true`;
+      connections[dbName] = dbc(connectionString);
+      let connection = connections[dbName];
+      
+export default class RawReportingEventsRepository {
+
+    queryRawEvents = async(query) => {

```

Comment:
'query' should get an appropriate type (you can even try to force the queries to follow the certain structure so 'select' first etc)
perhaps there are some type definitions in 'pg' library or in additional packages - you can also create one yourself